src/main/java/org/folio/circulation/domain/notice/schedule/ScheduledDigitalReminderHandler.java

```diff
@@ -159,7 +160,9 @@ private CompletableFuture<Result<ScheduledNoticeContext>> updateLoan(ScheduledNo
     }
 
     return loanRepository.updateLoan(
-      context.getLoan().withIncrementedRemindersLastFeeBilled(systemTime))
+      context.getLoan()
+        .withIncrementedRemindersLastFeeBilled(systemTime)
+        .withAction(LoanAction.REMINDER_FEE.getValue()))
```

Comment:
If withAction() method accepted LoanAction as a parameter, we wouldn't need to call getValue() here?